src/components/Profile/Collected.tsx

```diff
@@ -55,7 +55,7 @@ const Collected = () => {
     if (address) {
       fetchMoreWikis()
     }
-  }, [address, fetchMoreWikis])
+  }, [address])
```

Comment:
Let's appease eslint
```suggestion
  // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [address])
```

Reply:
alright